### DISCUSSION TITLE
Potential talks/discussions at Oct LinkedMusic Project Meeting

### DISCUSSION THREAD

**Discussion by dchiller** [DDMAL/linkedmusic-datalake#19] [General]
These will probably have lots of discussion and feedback time. Prepare for about 15 minutes.

- Introduction to LinkedMusic Project (@fujinaga)
- Introduction to semantic web: RDF, SPARQL, Wikidata, JSON-LD, etc. (@fujinaga) 
- General Wikidata and Openrefine overview (@maregold)
- Getting database extracts reconciled and in JSON-LD (@malajvan)
- Dummy search box with explanation of what works and what doesn't: solr, elastic search, etc. (@dchiller @homework36)
- Music instruments on Wikidata (@kyrieb-ekat)
- VIM demo (@yinanazhou)
- Next steps for VIM (@dchiller)
- SIMSSA DB (@rebmizrahi @codaich)
- Cantus DB: Export for the data lake (@jacobdgm)

Also: prepare topics and questions for Technical Session on Friday, 20 Oct (10am - 12pm)

**Comment by malajvan** [comment]
Hi, Does anyone remember when we decided to meet to go over the presentations before the Project Meeting weekend?

**Reply by dchiller** [reply]
I did not remember setting one, so I could also use the reminder.

**Reply by maregold** [reply]
Friday, 13 October from 10-12 pm!

**Reply by malajvan** [reply]
Thank you!

**Reply by fujinaga** [reply]
It will be in the lab, not in the Conference room.

**Comment by annamorphism** [comment]
DRAFT AGENDA FOR PROJECT MEETING
@jacobdgm and @yinanazhou take particular note that this is not the same as the first emailed draft.
@dchiller @homework36 note that the very last presentations on Saturday could end up being on Sunday if discussion spills over. 

## Saturday 21 October 2023

### 09:00–10:30 Introductions and databases
- Introductions: Linked Data, RDF, SPARQL, and JSON-LD (Ichiro Fujinaga)
- Wikidata and OpenRefine (Marisa Goldman)
- MetaBrainz (Alastair Porter)
- SIMSSA DB (Cory McKay / Rebecca Mizrahi)

### 10:30–11:00 Coffee Break

### 11:00–12:30 Databases
- Cantus DB (Debra Lacoste / Jacob deGroot-Maggetti)
- Cantus Ultimus (Dylan Hillerbrand)
- RISM Digital (Laurent Pugin)
- DIAMM (Andrew Hankinson)

### 12:30–13:30 Lunch Break

### 13:30–15:00 Current projects and demonstrations
- TROMPA experience (Alastair Porter)
- Introduction to current projects (Ichiro Fujinaga)
- Virtual Instrument Museum (VIM) Demonstration (Yinan Zhou)
- Musical Instruments in Wikidata (Kyrie Bouressa)
- Data Lake Demonstration (Dylan Hillerbrand) 
- Data export for the Data Lake (Hong Van Pham / Jacob deGroot-Maggetti)

### 15:00–15:30 Coffee Break

### 15:30–17:00 Other business and project development
- Formation of LinkedMusic Project Committees
- Suggestions for future LinkedMusic guest speakers
- Addition of other databases and projects
- Experiments with Solr (Dylan Hillerbrand)
- Experiments with ElasticSearch (Hanwen Zhang)

## Sunday 22 October 2023

### 9:00–10:30 Technical Discussions I
- Virtual Instruments Museum

### 10:30–11:00 Coffee Break

### 11:00–12:30 Technical Discussions I
- Data Lake

**Comment by malajvan** [comment]
Also is there a general rule/template for the presentation slides? I'm unsure where to find some of the lab's presentations

**Reply by fujinaga** [reply]
There are PDFs of presentations from last year by most people here: https://linkedmusic.ca/activities/
Here's the PowerPoint version of my Keynote presentation I gave last year.
[fujinaga22LM_ProjectMeeting.pptx](https://github.com/DDMAL/linkedmusic-datalake/files/12872455/fujinaga22LM_ProjectMeeting.pptx)

**Comment by malajvan** [comment]
Another question: Can we use our own computer for the presentation (also to do demo if need to) or do we have to setup everything on a single laptop? And if we can use our own, does the lab have a usbc to HDMI converter?

**Comment by fujinaga** [comment]
On Oct 12, 2023, at 13:47, Van Pham ***@***.***> wrote:

﻿

Another question: Can we use our own computer for the presentation (also to do demo if need to) or do we have to setup everything on a single laptop?

Your own.

And if we can use our own, does the lab have a usbc to HDMI converter?

I have one, so you can use that.

**Reply by malajvan** [reply]
That's great! Can you bring it tomorrow for the presentation run through? Thank you very much!
